### ISSUE TITLE
nutanix_subnet SIGSEGV on network rename

### ISSUE THREAD

**Issue by rafaldutko-ig** [nutanix/terraform-provider-nutanix#118]
### Nutanix Cluster Information
Version 5.11 - Starter License
NCC Version: 3.8.0.1
LCM Version: 2.2.9803 

### Terraform Version
Terraform v0.12.24
+ provider.nutanix v1.0.2
also checked on
Terraform v0.11.14
+ provider.nutanix v1.0.2

### Affected Resource(s)
* nutanix_subnet

### Terraform Configuration Files
```hcl
data "nutanix_clusters" "clusters" {
}

output "cluster" {
  value = data.nutanix_clusters.clusters.entities.0.metadata.uuid
}

resource "nutanix_subnet" "next-iac-managed" {
  # What cluster will this VLAN live on?
  cluster_uuid = data.nutanix_clusters.clusters.entities.0.metadata.uuid

  # General Information
  name        = "next-iac-managed-kotki"
  vlan_id     = 101
  subnet_type = "VLAN"

}
```
### Debug Output
https://gist.github.com/rafaldutko-ig/2465e7f1c382a772a05a94b7f2889cec
### Panic Output
https://gist.github.com/rafaldutko-ig/b664c3a7cd396e6055ae7beed1a35a6e
### Expected Behavior
Network renamed from next-iac-managed-example to next-iac-managed-kotki
### Actual Behavior
Terraform crashed with SIGSEGV
### Steps to Reproduce
1. `terraform apply`
### Important Factors
Using a Prism Element cluster
​
### References

* #0000

**Comment by PacoDw** [comment]
Hello @rafaldutko-ig thank you so much for your review, I'm working on it, I'll let you know when I have news

**Comment by PacoDw** [comment]
The issue is fixed you can test it on the fix-#118 branch

Let me know if you have another comment or concern, thanks
